CONTRIBUTING.md

```diff
@@ -149,6 +149,19 @@ CREATE TABLE [dbo].[HCRWriteData](
 	[b] [float] NULL,
 	[c] [varchar](255) NULL
 )
+
+CREATE TABLE [dbo].[dermatologyDeployClassificationBASE](
```

Comment:
I had to do `[SAM].[dbo].[dermatologyDeployClassificationBASE]` but I'm not sure what that's about. The way you have it is consistent with other examples.